osmaxx-py/osmaxx/contrib/auth/migrations/0002_add_default_usergroup_osmaxx.py

```diff
@@ -8,7 +8,13 @@
 def update_site_forward(apps, schema_editor):
     """Add group osmaxx."""
     Group = apps.get_model("auth", "Group")
-    Group.objects.create(name=settings.OSMAXX_FRONTEND_USER_GROUP)
+    Permission = apps.get_model("auth", "Permission")
+    ContentType = apps.get_model("contenttypes", "ContentType")
+    ExtractionOrder = apps.get_model("excerptexport", "ExtractionOrder")
+    group = Group.objects.create(name=settings.OSMAXX_FRONTEND_USER_GROUP)
+    content_type = ContentType.objects.get_for_model(ExtractionOrder)
```

Comment:
As we get **or create** a permission, should we remove it again in the backwards migration? (Probably not, as it'd be recreated by the `post_migrate` signal handler, anyway.)